centrallix/osdrivers/objdrv_dbl.c

```diff
@@ -1654,13 +1654,15 @@ dbl_internal_ParseColumn(pDblColInf column, pObjData pod, char* data, char* row_
 		f = 1;
 		for(i=0;i<column->DecimalOffset;i++) f *= 10;
 		pod->Money = (pMoneyType)data;
-		pod->Money->WholePart = v/f;
+		pod->Money->Value = (v/f) * 10000;
```

Comment:
tests?

Reply:
There should be tests for this, but they can probably be based off the objdrv_fp tests